release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1727,10 +1732,55 @@ If these values are not defined in a failure domain configuration, for instance
+
+* Previously, API calls to Cincinnati that encountered errors did not fail immediately, potentially leading to inconsistencies. With this update, errors related to Cincinnati are now considered severe and cause an immediate failure. (link:https://issues.redhat.com/browse/OCPBUGS-38461[*OCPBUGS-38461*])
+
+* Previously, the release signature `ConfigMap` for oc-mirror plugin v2 was incorrectly stored in an archived TAR file instead of in the `cluster-resources` folder. This caused `mirror2disk` to fail. With this release. the release signature `ConfigMap` for oc-mirror plugin v2 that is in JSON format or YAML format, compatible with  oc-mirror plugin v1, now get stored in the `cluster-resources` folder. (link:https://issues.redhat.com/browse/OCPBUGS-38343[*OCPBUGS-38343*]) and (link:https://issues.redhat.com/browse/OCPBUGS-38233[*OCPBUGS-38233*])
+
+* Previously, when running oc-mirror v2 Technology Preview in mirror-to-disk mode within an disconnected environment, the tool attempted to download `graph.tar.gz` from `api.openshift.com`, causing the mirroring process to fail. With this fix, when the `UPDATE_URL_OVERRIDE` environment variable is set, oc-mirror first checks for the graph image in the oc-mirror cache. If the image is not found, oc-mirror skips it without failing, ensuring successful mirroring in air-gapped environments. (link:https://issues.redhat.com/browse/OCPBUGS-38037[*OCPBUGS-38037*])
+
+* Previously, oc-mirror v2 Technology Preview did not support images referenced by both `tag` and `digest`. These images were excluded from the archive during the mirror-to-disk workflow, causing failures when mirroring from the archive to a disconnected mirror registry. With this fix, oc-mirror v2 includes these images in the archive by pulling the source image by `digest` while retaining the `tag` for reference. A warning message is displayed to inform users of this behavior.  (link:https://issues.redhat.com/browse/OCPBUGS-37867[*OCPBUGS-37867*])
+
+* Previously, using an invalid log-level flag caused oc-mirror plugin v2 to panic. This update ensures that oc-mirror plugin v2 handles invalid log levels gracefully. Additionally, the `loglevel` flag has been renamed to `log-level` to align with tools like Podman. (link:https://issues.redhat.com/browse/OCPBUGS-37740[*OCPBUGS-37740*])
+
+
 [discrete]
 [id="ocp-release-note-openshift-cli-bug-fixes_{context}"]
 ==== OpenShift CLI (oc)
 
+* Previously, the `oc adm node-image create --pxe generated` command did not create only the Preboot Execution Environment (PXE) artifacts. Instead, the command created the PXE artifacts with other artifacts from a `node-joiner` pod and stored them all in the wrong subdirectory. Additionally, the PXE artifacts were incorrectly prefixed with `agent` instead of `node`. With this release, generated PXE artifacts are stored in the correct directory and receive the correct prefix. (link:https://issues.redhat.com/browse/OCPBUGS-46449[*OCPBUGS-46449*])
```

Comment:
Does the command still generate the extraneous artifacts from the `node-joiner` pod?

Reply:
I cannot answer this. The writer will need to confirm.